calico_kubernetes/calico_kubernetes.py

```diff
@@ -414,10 +416,10 @@ def _generate_rules(self, pod):
         annotations = self._get_metadata(pod, "annotations")
 
         # Find policy block of annotations
-        if annotations and "policy" in annotations.keys():
+        if annotations and POLICY_ANNOTATION_KEY in annotations.keys():
```

Comment:
Another little nitpick.  You can check for the existence of a key in a dictionary without calling .keys().

This will do the same thing, without the extra work of calling keys()
```
if annotations and POLICY_ANNOTATION_KEY in annotations:
  ...
```